snapd-glib/snapd-error.h

```diff
@@ -62,6 +62,7 @@ G_BEGIN_DECLS
  * @SNAPD_ERROR_NOT_A_SNAP: the given snap or directory does not look like a snap.
  * @SNAPD_ERROR_DNS_FAILURE: A hostname failed to resolve during the request.
  * @SNAPD_ERROR_OPTION_NOT_FOUND: A requested configuration option is not set.
+ * @SNAPD_ERROR_UNSUCCESSFUL: A snapctl command was unsuccessful.
```

Comment:
Does this error only apply to snapctl? The name of it suggests it could be due to anything.

Reply:
I copied the description from the corresponding constant in snapd:

https://github.com/snapcore/snapd/blob/9221806b95d60aab3e981b590c8440691ca1a898/client/errors.go#L124-L125

It is currently only used by the snapctl API, so it is accurate at present.